2/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM quay.io/openshift/origin-cli
+FROM openshift/origin:v3.11
+#FROM quay.io/openshift/origin-cli
```

Comment:
why is this changing?  (also referencing v3.11 seems odd)

Reply:
ah ... that was suppose to be temporary as I didn't want to wrestle with the UBI stuff as well ... will revert